### PR TITLE
Card

### PR DESCRIPTION
**Pull request by arpit01923**
- Different card variation are created

1. Card with Badge
2. Card with dismiss
3. Card with text-overlay
4. Text only Card
5. Horizontal card
6. Card with shadow
image

### CODE REVIEW

File: css/components/Card.css [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+.horizontal-footer{
+    order: 4;
+}
+
+/* Card with shadow */
+
+.shadow{
+    box-shadow: 15px 15px 10px;
+}
```
Comment (bajajcodes) [inline css/components/Card.css:159]: add new line at end here,

File: css/components/Card.css [hunk 2]
```diff
@@ -0,0 +1,159 @@
+@import "../style.css";
+
+body{
```
Comment (bajajcodes) [inline css/components/Card.css:3]: Why this piece of code here, it's to be in `style.css` if I am not wrong.

File: css/components/Card.css [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+    display: flex;
+    flex-wrap: wrap;
+}
+
+/* Card with Badge */
+.badge, .dismiss, .text-overlay, .text, .horizontal, .shadow{
```
Comment (bajajcodes) [inline css/components/Card.css:9]: Compoent base style under class `name card` may have easy relatibility.

File: css/components/Card.css [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+    position: relative;
+    width: 200px;
+    height: 270px;
+    border: 1px solid var(--badge-offline);
+    background-color: var(--badge-offline);
+    border-radius: 1rem;
+    margin: .5rem;
+}
+
+.badge .far, .dismiss .far, .text-overlay .far, .text .far, .horizontal .far, .shadow .far{
```
Comment (bajajcodes) [inline css/components/Card.css:21]: Suggestion: class name `far` isn't conveving its meaning.

Comment (bajajcodes) [reply css/components/Card.css:21]: If you add explanation or description comments here it would be good.

File: css/components/Card.css [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+
+.horizontal .far{
+    display: none;
+}
+
+.horizontal-header{
+    order: 2;
+}
+
+.horizontal-header h2, .horizontal-header h3{
```
Comment (bajajcodes) [inline css/components/Card.css:136]: It can better if givenc class names to element h2 and h3.

**Review by bajajcodes — COMMENTED** [review]
- If possible again think about css selectors
- Give class name to elements and then use to apply css instead of combination of class-name and element.
- Describe your code what trying to achieve.

File: css/components/Card.css (hunk 2 above)
Comment (Arpit-dotcom) [reply css/components/Card.css:3]: made a container for it

File: css/components/Card.css (hunk 3 above)
Comment (Arpit-dotcom) [reply css/components/Card.css:9]: conflict resolved

File: css/components/Card.css (hunk 4 above)
Comment (Arpit-dotcom) [reply css/components/Card.css:21]: Changed name to badge

File: css/components/Card.css (hunk 5 above)
Comment (Arpit-dotcom) [reply css/components/Card.css:136]: class name updated

File: css/components/Card.css (hunk 1 above)
Comment (arpit01923) [reply css/components/Card.css:159]: done

File: css/components/Card.css (hunk 2 above)
Comment (arpit01923) [reply css/components/Card.css:3]: added it to a div container

File: css/components/Card.css (hunk 3 above)
Comment (arpit01923) [reply css/components/Card.css:9]: conflict resolved

File: css/components/Card.css (hunk 4 above)
Comment (arpit01923) [reply css/components/Card.css:21]: change class name to badge

File: css/components/Card.css (hunk 5 above)
Comment (arpit01923) [reply css/components/Card.css:136]: class name updated

**Pull request merged**